src/SpikeCore/SpikeCore.Web/Program.cs

```diff
@@ -4,35 +4,49 @@
 
 using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+
+using Rebus.Activation;
+using Rebus.Config;
+using Rebus.Transport.InMem;
 
 namespace SpikeCore.Web
 {
     public class Program
     {
         public static async Task Main(string[] args)
         {
-            var webHost = WebHost
-                .CreateDefaultBuilder(args)
-                .UseStartup<Startup>()
-                .Build();
+            using (var activator = new BuiltinHandlerActivator())
```

Comment:
Would you mind adding a comment up here explaining that we have to do the wiring in our Main block due to the way that Rebus works? I would normally expect this to be in the Core module.